### PR TITLE
Travis-ci: added support for ppc64le

### PR DESCRIPTION
**Pull request by dthadi3**
Signed-off-by: Devendranath Thadi <devendranath.thadi3@gmail.com>

Added power support for the travis.yml file with ppc64le. This is part of the Ubuntu distribution for ppc64le. This helps us simplify testing later when distributions are re-building and re-releasing.

### CODE REVIEW

**Comment by dthadi3** [comment]
@hush-hush  waiting for your response for merging code ppc64le. Thanks!

File: .travis.yml [hunk 1]
```diff
@@ -35,5 +55,6 @@ jobs:
 script:
   # race detector is only available on amd64
  - if [[ "$TRAVIS_CPU_ARCH" == "amd64" ]]; then go test -race -v ./...; else go test -v ./...; fi
+ - if [[ "$TRAVIS_CPU_ARCH" == "ppc64le" ]]; then go test -race -v ./...; else go test -v ./...; fi
```
Comment (hush-hush) [inline .travis.yml:58]: Could we meged this with the previous if ?

**Review by hush-hush — CHANGES_REQUESTED** [review]
On tiny detail and we could merge.

File: .travis.yml (hunk 1 above)
Comment (dthadi3) [reply .travis.yml:58]: yeah sure, Below one is fine?

if [[ "$TRAVIS_CPU_ARCH" == "amd64" ]] || [[ "$TRAVIS_CPU_ARCH" == "ppc64le" ]]; then go test -race -v ./...; else go test -v ./...; fi

**Comment by hush-hush** [comment]
Thanks for this PR @dthadi3

**Pull request merged**